settings.py

```diff
@@ -833,6 +833,7 @@
     'custom.cowin.repeaters.BeneficiaryRegistrationRepeater',
     'custom.cowin.repeaters.BeneficiaryVaccinationRepeater',
     'corehq.motech.repeaters.expression.repeaters.CaseExpressionRepeater',
+    'corehq.motech.repeaters.expression.repeaters.ArcGISFormExpressionRepeater',
```

Comment:
Maybe out of scope for this PR, but we should think of adding the FormExpressionRepeater here as well now that it exists, since users can technically start using that for more than just ArcGIS integrations.

Reply:
That is a great point. I think it falls a bit out of scope here since a bit more work and testing will need to be done for the Form Expression repeater, however I totally agree that this is a worthwhile inclusion. I'll create a separate ticket so that this can be added in at a later point.

Reply:
as a GA?

Reply:
No I think this should perhaps be put behind the same feature flag as the configurable case repeater (or it's own one). This will need to be determined when discussing the ticket, but I don't think this should be a GA yet.

Reply:
I think we can put it behind the `EXPRESSION_REPEATER` feature flag, the same as CaseExpressionRepeater.